app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name, health=100, hidden=False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        a = f"Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}"
+        return f'{{{a}}}'
```

Comment:
what is `a`?

Reply:
Removed this crutch

Reply:
Line break error 12, cannot break a line correctly